3_navigation.ipynb

```diff
@@ -14,18 +16,54 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "# Localization and autonomous movement\n",
+    "\n",
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Objective</b>\n",
+    "    <p>In this notebook, we will learn how to move the robot autonomously to the goal while avoiding the objects.</p>\n",
```

Comment:
Use this sentence if the goal is already known to the reader from prior context:

    "    <p>In this notebook, we will learn how to move the robot autonomously to the goal while avoiding other objects.</p>\n",

If the goal has not been introduced, yet, then use this sentence:

    "    <p>In this notebook, we will learn how to move the robot autonomously to a goal while avoiding other objects.</p>\n",